package.json

```diff
@@ -2,26 +2,19 @@
   "name": "dota2",
   "version": "0.7.1",
   "dependencies": {
-    "protobuf": "*",
-    "deferred": "*",
-    "bignumber.js": "2.0.5",
-    "merge": ">= 0.0.2"
-  },
-  "peerDependencies": {
-    "steam": ">= 0.6.6"
+    "bignumber.js": "2.0.7",
+    "deferred": "^0.7.2",
+    "merge": "^1.2.0",
+    "protobufjs": "^4.0.0",
+    "steam": "^1.0.0"
   },
   "readmeFilename": "README.md",
   "description": "A node-steam plugin for Dota 2.",
   "main": "index.js",
-  "repository": {
```

Comment:
The repository is used for the NPM page: https://www.npmjs.com/package/dota2

Not sure what author is used for.  I'm not too fussed about being listed there, I'm out of touch with things now. Sadly I've been too busy over the past year or so to keep up with the maintenance of this library.  Jimmy's been doing a great job keeping on top of things though. :)